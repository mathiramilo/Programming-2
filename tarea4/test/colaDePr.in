# Prueba de operaciones de colaDePrioridad
insertarEnCP 5 15.0
insertarEnCP 99 1.5
insertarEnCP 7 120.3
insertarEnCP 32 1.6
insertarEnCP 21 -0.9
insertarEnCP 23 50.4
insertarEnCP 19 36.2
insertarEnCP 31 1.9
insertarEnCP 17 2.3
insertarEnCP 65 -5.4
insertarEnCP 63 2.9
insertarEnCP 91 8.3
insertarEnCP 13 4.1
insertarEnCP 16 5.3
insertarEnCP 46 8.7
insertarEnCP 35 40.9
insertarEnCP 2 35.6
insertarEnCP 75 13.3
insertarEnCP 74 -3.5
prioritario
eliminarPrioritario
prioritario
eliminarPrioritario
prioritario
eliminarPrioritario
prioritario
eliminarPrioritario
prioritario
eliminarPrioritario
prioritario
eliminarPrioritario
actualizarEnCP 35 2.5
prioritario
eliminarPrioritario
prioritario
prioridad 35
actualizarEnCP 19 -10.0
prioritario
eliminarPrioritario
estaEnCP 19
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
eliminarPrioritario
estaVaciaCP
Fin
