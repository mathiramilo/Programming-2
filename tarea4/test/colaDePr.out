1>#  Prueba de operaciones de colaDePrioridad.
2>Se insertó el elemento.
3>Se insertó el elemento.
4>Se insertó el elemento.
5>Se insertó el elemento.
6>Se insertó el elemento.
7>Se insertó el elemento.
8>Se insertó el elemento.
9>Se insertó el elemento.
10>Se insertó el elemento.
11>Se insertó el elemento.
12>Se insertó el elemento.
13>Se insertó el elemento.
14>Se insertó el elemento.
15>Se insertó el elemento.
16>Se insertó el elemento.
17>Se insertó el elemento.
18>Se insertó el elemento.
19>Se insertó el elemento.
20>Se insertó el elemento.
21>65
22>Eliminado.
23>74
24>Eliminado.
25>21
26>Eliminado.
27>99
28>Eliminado.
29>32
30>Eliminado.
31>31
32>Eliminado.
33>Se actualizó el valor.
34>17
35>Eliminado.
36>35
37>2.50.
38>Se actualizó el valor.
39>19
40>Eliminado.
41>No está.
42>Eliminado.
43>Eliminado.
44>Eliminado.
45>Eliminado.
46>Eliminado.
47>Eliminado.
48>Eliminado.
49>Eliminado.
50>Eliminado.
51>Eliminado.
52>Eliminado.
53>Vacia.
54>Fin.
